website/project/views/drafts.py

```diff
@@ -128,11 +128,12 @@ def make_draft_registraton(user, node, schema, data):
         registration_schema=schema,
         registration_metadata=data,
     )
-    if schema.requires_approval:
-        draft.approval = DraftRegistrationApproval(
-            initiated_by=user,
-            end_date=None,  # TODO: expire me?
-        )
+    # TODO: implement in schema first
```

Comment:
Both this and the block below were throwing errors every time I opened the editor. I assumed I didn't have the relevant schema code for requires approval